measure/src/main/scala/org/apache/griffin/measure/Application.scala

```diff
@@ -63,10 +63,10 @@ object Application extends Loggable {
     val allParam: GriffinConfig = GriffinConfig(envParam, dqParam)
 
     // choose process
-    val procType = ProcessType(allParam.getDqConfig.getProcType)
+    val procType = ProcessType.withNameWithDefault(allParam.getDqConfig.getProcType)
     val dqApp: DQApp = procType match {
-      case BatchProcessType => BatchDQApp(allParam)
-      case StreamingProcessType => StreamingDQApp(allParam)
+      case ProcessType.BatchProcessType => BatchDQApp(allParam)
+      case ProcessType.StreamingProcessType => StreamingDQApp(allParam)
```

Comment:
example: wildcard import can eliminate this change.

Reply:
I made the changes, please review it.